functions/src/api/v1/rooms/createRoom.ts

```diff
@@ -0,0 +1,60 @@
+import { Request, Response } from 'express'
+import { addRoom } from '../../../db/addRoom'
+import { createTwilioRoom } from './service/createTwilioRoom'
+import { updateRoom } from '../../../db/updateRoom'
+import { RoomId, RoomSid } from '../../../types/Room'
+
+export interface NewRoomResponse {
+    roomId: RoomId
+    roomSid: RoomSid
```

Comment:
what does the `roomSid` stand for ?

Reply:
this is the twilio room sid that will maybe be needed later on

Reply:
The difference being ? The `id` is the one we create, the `Sid` is the one created by the provider (twilio in this case) ?

Reply:
yes exactly